src/extension/index.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('activate', () => {
     before(async () => {
         const { activate } = proxyquire('.', {
             'fs': {
-                '@global': true,
+                // '@global': true,
```

Comment:
>// '@global': true, [](http://example.com/codeflow?start=16&length=19)

Do we need to keep this? (I saw others too)

Reply:
Right...I can remove this one. I'll see if I can remove the others. Best if we can avoid `@global`.

Reply:
Just verified...it needs to stick around for the `vscode` imports. All others can be removed.